### PR TITLE
ANC Logic - diagnosis Preeclampsia

### PR DESCRIPTION
**Pull request by anvmn**
#139 
#182

### CODE REVIEW

**Comment by anvmn** [comment]
@mariano-dagostino please review

**Comment by anvmn** [comment]
@nedSaf please review

File: client/src/elm/Pages/Prenatal/Activity/Utils.elm [hunk 1]
```diff
@@ -318,16 +319,19 @@ expectNextStepsTask currentDate assembled task =
...
                    )
 
         NextStepsWait ->
-            -- We show Wait activity when there's at least one
-            -- test that was performed, or, 2 hours waiting is
-            -- required for blood preasure recheck.
-            getMeasurementValueFunc assembled.measurements.labsResults
-                |> Maybe.map .performedTests
-                |> isJust
+            -- If we reffer patient somewhere, there's no need to wait.
```
Comment (nedSaf) [inline client/src/elm/Pages/Prenatal/Activity/Utils.elm]:
```suggestion
            -- If we refer patients somewhere, there's no need to wait.
```

File: client/src/elm/Pages/Prenatal/Activity/Utils.elm [hunk 2]
```diff
@@ -978,44 +998,128 @@ matchExaminationPrenatalDiagnosis egaInWeeks measurements diagnosis =
...
 
-immediateHypertensionCondition : Float -> Float -> Bool
-immediateHypertensionCondition dia sys =
-    dia >= 110 || sys >= 160
-
-
-{-| We measure BP again when we suspect Hypertension (dia between 90
-and 110, and dia between 140 and 160).
+{-| We measure BP again when we suspect Hypertension or Preeclamsia
+(dia between 90 and 110, and dia between 140 and 160).
```
Comment (nedSaf) [inline client/src/elm/Pages/Prenatal/Activity/Utils.elm:1065]: `dia`? Just making sure it's not a typo.

**Review by nedSaf — APPROVED** [review]
👍  Looks good, one minor typo.

File: client/src/elm/Pages/Prenatal/Activity/Utils.elm (hunk 2 above)
Comment (nedSaf) [reply client/src/elm/Pages/Prenatal/Activity/Utils.elm:1065]: Ohh I see what's dia after checking the code below...

**Pull request merged**